pvlib/modelchain.py

```diff
@@ -385,6 +412,37 @@ def __setattr__(self, key, value):
             value = self._result_type(value)
         super().__setattr__(key, value)
 
+    def __repr__(self):
+        # once per MC
+        mc_attrs = ['weather', 'solar_position', 'airmass', 'tracking', 'aoi',
+                    'aoi_modifier', 'total_irrad', 'spectral_modifier',
+                    'effective_irradiance', 'cell_temperature', 'diode_params',
+                    'dc', 'dc_ohmic_losses', 'losses', 'ac']
+
+        def _head(obj):
+            try:
+                return obj[:3]
+            except:
+                return obj
+
+        if type(self.dc) is tuple:
+            num_arrays = len(self.dc)
+        else:
+            num_arrays = 1
+
+        desc1 = ('=== ModelChainResult === \n')
+        desc2 = (f'Number of Arrays: {num_arrays} \n')
+        attr = 'times'
+        desc3 = ('Times (first 3)\n' +
+                 f'{_head(_getmcattr(self, attr))}' +
```

Comment:
```suggestion
                 f'{self.times[:3]}' +
```

Seems like this should be fine as long as it's safe to assume `self.times` is a `DatetimeIndex`

Reply:
Won't work if `times` is a DatetimeStamp. Maybe that can't happen, idk.